netlify/functions/submit-finding.ts

```diff
@@ -248,35 +252,54 @@ exports.handler = async (event) => {
   }
 
   const owner = process.env.GITHUB_CONTEST_REPO_OWNER;
-  const riskCode = getRiskCodeFromGithubLabel(risk);
 
   try {
-    const markdownPath = `data/${attributedTo}-${riskCode}.md`;
-    const qaOrGasSubmissionBody = `See the markdown file with the details of this report [here](https://github.com/${owner}/${repo}/blob/main/${markdownPath}).`;
-    const isQaOrGasSubmission = Boolean(riskCode === "G" || riskCode === "Q");
-    if (isQaOrGasSubmission) {
-      const existingReport = await getMarkdownReportForUser(
-        octokit,
-        repo,
-        attributedTo,
-        riskCode as "Q" | "G"
-      );
-      if (existingReport) {
-        throw {
-          status: 400,
-          message: `It looks like you've already submitted a ${risk} report for this contest.`,
-        };
+    let isQaOrGasSubmission = false;
+    let qaOrGasSubmissionBody = '';
+    let markdownPath = '';
+    let riskCode = '';
+    if(!isMitigationReport){
+      const riskCode = getRiskCodeFromGithubLabel(risk);
+      markdownPath = `data/${attributedTo}-${riskCode}.md`;
+      qaOrGasSubmissionBody = `See the markdown file with the details of this report [here](https://github.com/${owner}/${repo}/blob/main/${markdownPath}).`;
+      isQaOrGasSubmission = Boolean(riskCode === "G" || riskCode === "Q");
+      if (isQaOrGasSubmission) {
+        const existingReport = await getMarkdownReportForUser(
+          octokit,
+          repo,
+          attributedTo,
+          riskCode as "Q" | "G"
+        );
+        if (existingReport) {
+          throw {
+            status: 400,
+            message: `It looks like you've already submitted a ${risk} report for this contest.`,
+          };
+        }
       }
     }
 
+    //if it's a confirmed mitigation then no title will be provided
+    let isConfirmedMitigation = false;
+    if(isMitigationReport && (
+      !body || !title || !risk
+    )){
+      isConfirmedMitigation = true;
+    }else if(
+      isMitigationReport
+    ){
+      labels.push(`bug`)
```

Comment:
Do you know if there's any reason why we need the bug label? It seems like we should be able to leave it off, but maybe something downstream depends on it.

If we do need the bug label, I can add it back on the frontend.

Reply:
@captainmangoC4 I just added it in here to keep it consistent with how it has been done, but I don't see why it needs to be here. So I will remove it from here.